### PR TITLE
Edit Tag

### PR DESCRIPTION
**Pull request by nbriannl**
#29 

Edit the specified tag in all contacts containing this tag with a new specified tag

Format:
`edit old/TAG new/TAG`

Example:
`edit old/CS1020 new/CS2010`

This feature is branched off the new way Delete Tag is implemented. Hence commits also include those from #65

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -52,15 +52,25 @@
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + PREFIX_EMAIL + "johndoe@example.com\n"
+            + "OR "
+            + "Edit the specified tag in all contacts containing this tag with a new specified tag "
+            + "Parameters: [old/TAG] [new/TAG] "
```
Comment (zacharytang) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: Should import constant prefixes from CliSyntax class here for consistency

**Review by zacharytang — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (nbriannl) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: Done.

**Review by zacharytang — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 2]
```diff
@@ -52,14 +54,23 @@
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + PREFIX_EMAIL + "johndoe@example.com\n"
+            + "OR "
+            + "Edit the specified tag in all contacts containing this tag with a new specified tag "
+            + "Parameters: " + PREFIX_OLD_TAG + "TAG " + PREFIX_NEW_TAG + "TAG"
```
Comment (April0616) [inline src/main/java/seedu/address/logic/commands/EditCommand.java:60]: need one more whitespace after the second "TAG"?

File: src/test/java/seedu/address/logic/commands/DeleteCommandTest.java [hunk 3]
```diff
@@ -31,7 +31,6 @@
 public class DeleteCommandTest {
 
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
-    private Model model2 = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```
Comment (April0616) [inline src/test/java/seedu/address/logic/commands/DeleteCommandTest.java:34]: why was here a 'model2'? strange

**Pull request merged**